ci-operator/step-registry/openshift/image-based/upgrade/seed/gather/lca/openshift-image-based-upgrade-seed-gather-lca-commands.sh

```
@@ -35,11 +64,12 @@ EOF
 
 chmod +x ${SHARED_DIR}/gather_seed_lca.sh
 
-echo "Transfering gather LCA script..."
-scp "${SSHOPTS[@]}" ${SHARED_DIR}/gather_seed_lca.sh $ssh_host_ip:$remote_workdir
+echo "Transferring gather LCA script..."
+retry "${MAX_RETRY_ATTEMPTS}" "${INITIAL_RETRY_DELAY}" scp "${SSHOPTS[@]}" ${SHARED_DIR}/gather_seed_lca.sh $ssh_host_ip:$remote_workdir
 
-echo "Gather target LCA..."
-ssh "${SSHOPTS[@]}" $ssh_host_ip "${remote_workdir}/gather_seed_lca.sh"
+echo "Gather seed LCA..."
+retry "${MAX_RETRY_ATTEMPTS}" "${INITIAL_RETRY_DELAY}" ssh "${SSHOPTS[@]}" $ssh_host_ip "${remote_workdir}/gather_seed_lca.sh"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)